components/script/dom/performance.rs

```diff
@@ -177,6 +177,10 @@ impl Performance {
 
     /// Remove a PerformanceObserver from the list of observers.
     pub fn remove_observer(&self, observer: &DOMPerformanceObserver) {
+        if self.pending_notification_observers_task.get() {
+            self.notify_observers();
```

Comment:
The spec does not mention anything about this case :\ . But I would assume that instead of notifying all observers, we should notify only the one being removed here.

Note that you'll need to make a copy of the observer, just like we do inside [notify_observers](https://github.com/servo/servo/blob/master/components/script/dom/performance.rs#L186) before calling `observer.notify()`.

Reply:
Alright, I'll fix that. Thanks for your input!